### ISSUE TITLE
Number of bytes mismatch when reading tflite schema

### ISSUE THREAD

**Issue by dhc1256436519** [tensorflow/tensorflow#14929]
I train a model outside and convert it into.lite format and import it to the iOS project，but I  encounter a problem below:why?
Loaded model resolved reporter tensorflow/contrib/lite/interpreter.cc:481 required_bytes != bytes (4 != 16)Tensor 88 is invalidly specified in schema.
Failed to construct interpreter

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by apollo-time** [comment]
I am also got the same error.
My model is defined by 
```python
def make_model(input, scope, output_count, reuse=None):
    net = input
    batch_norm_params = {
        # Decay for the moving averages.
        'decay': 0.995,
        # epsilon to prevent 0s in variance.
        'epsilon': 0.001,
        # calculate moving average or using exist one
        'is_training': False,
        'variables_collections': [ tf.GraphKeys.TRAINABLE_VARIABLES ],
        'updates_collections': None,
    }
    with tf.variable_scope(scope, [input], reuse=reuse) as sc:
        with slim.arg_scope([slim.conv2d, slim.fully_connected],
                            activation_fn=tf.nn.relu,
                            weights_initializer=tf.truncated_normal_initializer(0.0, 0.01),
                            weights_regularizer=slim.l2_regularizer(0.00005),
                            biases_initializer=tf.constant_initializer(),
                            normalizer_fn=slim.batch_norm,
                            normalizer_params=batch_norm_params):
            with slim.arg_scope([slim.batch_norm], **batch_norm_params):
#                   net = slim.conv2d(net, 128, input.get_shape()[1:3], padding='SAME', scope='conv1')
                net = slim.conv2d(net, 32, 3, padding='VALID', scope='conv1')
                net = slim.max_pool2d(net, 3, 2, scope='pool1')
                net = slim.conv2d(net, 64, 3, padding='VALID', scope='conv2')
                net = slim.max_pool2d(net, 3, 2, scope='pool2')
                net = slim.conv2d(net, 64, 3, padding='VALID', scope='conv3')
                net = slim.max_pool2d(net, 2, 2, scope='pool2')
                net = slim.conv2d(net, 128, 2, padding='VALID', scope='conv4')
                net = slim.flatten(net)
                net = slim.fully_connected(net, 256, scope='fc1')
                label = slim.fully_connected(net, output_count, activation_fn=None, scope='fc_label')
                offset = slim.fully_connected(net, 2, activation_fn=None, scope='fc_offset')
    return label, offset
```
and input has shape [1,48,48,3]
I got error "tensorflow/contrib/lite/interpreter.cc:481 required_bytes != bytes (4 != 8)Tensor 28 is invalidly specified in schema.\ntensorflow/contrib/lite/interpreter.cc:481 required_bytes != bytes (4 != 8)Tensor 38 is invalidly specified in schema.\n", when in tflite::InterpreterBuilder function.

**Comment by apollo-time** [comment]
@dhc1256436519 Did you resolve this issue?

**Comment by tensorflowbutler** [comment]
Nagging Assigneee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by andrehentz** [comment]
Is this still a problem for you?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 31 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
